x/evm/module.go

```diff
@@ -58,7 +62,7 @@ func (AppModuleBasic) RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 
 // ConsensusVersion returns the consensus state-breaking version for the module.
 func (AppModuleBasic) ConsensusVersion() uint64 {
-	return 5
```

Comment:
Why did the version go up?

Reply:
https://github.com/xpladev/ethermint/pull/14/commits/5829d3092507eb853b29c69802adc981a3b043fd